spark/src/test/scala/org/apache/spark/sql/CometTPCDSQuerySuite.scala

```diff
@@ -157,9 +191,11 @@ class CometTPCDSQuerySuite
     conf.set(CometConf.COMET_EXEC_ENABLED.key, "true")
     conf.set(CometConf.COMET_EXEC_ALL_OPERATOR_ENABLED.key, "true")
     conf.set(CometConf.COMET_EXEC_SHUFFLE_ENABLED.key, "true")
-    conf.set(CometConf.COMET_MEMORY_OVERHEAD.key, "20g")
+    conf.set(CometConf.COMET_MEMORY_OVERHEAD.key, "15g")
+    conf.set(CometConf.COMET_SHUFFLE_ENFORCE_MODE_ENABLED.key, "true")
+    conf.set("spark.sql.adaptive.coalescePartitions.enabled", "true")
```

Comment:
Before we can close https://github.com/apache/datafusion-comet/issues/387 we should either change the default for `COMET_SHUFFLE_ENFORCE_MODE_ENABLED` or remove it completely.

Reply:
This can be a separate PR but we should not close the issue when we merge this PR